### DISCUSSION TITLE
Modeling reverse current stimulation

### DISCUSSION THREAD

**Discussion by wys5188** [simnibs/simnibs#333] [8. Simulation workflows] [answered]
Dear Experts, with Simnibs there a way to account for reversing the current of stimulation (see blue highlight in picture below)?

<img width="1000" alt="image" src="https://github.com/simnibs/simnibs/assets/17935441/7376daab-89e0-4a25-ae72-39f7514d4aa9">

Some of our colleagues have negated dI/dt to account for this in their E-field modeling toolbox. However, the output values for dI/dt in Localite are still positive so we aren't sure if this is what we should be doing with Simnibs.

**Comment by TorgeW** [comment, marked as answer]
When using scripting you can set dI/dt to a negative value to invert the field (reverse the current direction).
You can test that by just running two simulations with the same setting and just change Di/dt and compare the results. The magnitude should be the same, the field should just be inverted.
